services/sonarr/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner. 
```

Comment:
What does this mean? / What is the intention to have this here?

Reply:
Fixed